jina/hubble/hubio.py

```diff
@@ -533,13 +533,13 @@ def _get_prettyprint_usage(self, console, executor_name, usage_kind=None):
     @disk_cache_offline(cache_file=str(_cache_file))
     def fetch_meta(
         name: str,
-        tag: Optional[str] = None,
+        tag: str,
         secret: Optional[str] = None,
         force: bool = False,
     ) -> HubExecutor:
         """Fetch the executor meta info from Jina Hub.
         :param name: the UUID/Name of the executor
-        :param tag: the version tag of the executor
+        :param tag: the tag of the executor if available, otherwise, use `None` as the value
```

Comment:
The comment should be that `tag` should be passed via `args` and `force` and `secret` as `kwargs`, otherwise `cache` does not work